hedera-node/configuration/dev/settings.txt

```diff
@@ -41,7 +41,6 @@ reconnect.asyncStreamTimeoutMilliseconds,      60000
 metrics.csvFileName,                           MainNetStats
 metrics.csvOutputFolder,                       data/stats
 showInternalStats,                             true
-prometheus.endpointEnabled,                    false                        # differs from mainnet
```

Comment:
I want the prometheus endpoint to run in dev, so we can poll it to know the status of the node.